src/renderer/terms.js

```diff
@@ -6,12 +6,18 @@ import { useSelector } from "react-redux";
 import { languageSelector } from "~/renderer/reducers/settings";
 import { urls } from "~/config/urls";
 
-const getRawLanguageURL = (language: string) =>
-  `https://raw.githubusercontent.com/LedgerHQ/ledger-live-desktop/master/TERM.${language}.md`;
 const rawDefaultURL =
   "https://raw.githubusercontent.com/LedgerHQ/ledger-live-desktop/master/TERMS.md";
+
 export const url = "https://github.com/LedgerHQ/ledger-live-desktop/blob/master/TERMS.md";
 
+const termsUrlLocalized = {
+  en: "https://raw.githubusercontent.com/LedgerHQ/ledger-live-desktop/master/TERMS.md",
+  fr: "https://raw.githubusercontent.com/LedgerHQ/ledger-live-desktop/master/TERM.fr.md",
+  es: "https://raw.githubusercontent.com/LedgerHQ/ledger-live-desktop/master/TERM.es.md",
+  ru: "https://raw.githubusercontent.com/LedgerHQ/ledger-live-desktop/master/TERM.ru.md",
```

Comment:
I'm not super fan of this idea that it won't scale each time we just add a TERM file. I actually wonder if we shouldn't have a "try load and fallback" approach instead.

it would be more future proof that any live version can be backward compatible to new TERM that would be added in future.

Reply:
the try load approach was the issue reported at first if i understood correctly :shrug: 
its still in place though i haven't changed it just made sure if we have the url ready we target it directly instead of 404 then going back to english.